examples/header-group.html

```diff
@@ -0,0 +1,80 @@
+<template id=header-group>
+
+<style>
+
+  header-group
+    { display: block }
+
+  header-group
+    hgroup
+      { background: #bada55; color: var(--darkest-color) }
+
+</style>
+
+<hgroup>
+  <h1><a href=/examples/index.html>snuggsi ツ</a></h1>
+  <slot name=header><h2>try an example</h2></slot>
+</hgroup>
+
+</template>
+
+<script>
+const tag = `header-group`
+
+Element `header-group`
+
+(class extends HTMLElement {
+
+  static onimportload (event) {
```

Comment:
@brandondees we should upstream `onimportload` to snuggsi. This way it "automagically" happens. Not sure if  `onimportload` is a proper name tho. Also not sure if `onload` would be monkeypatching some other existing event either.